### PR TITLE
fix issues in picture_id populating

### PR DESCRIPTION
**Pull request by taste1981**

### CODE REVIEW

File: modules/video_coding/timing.cc [hunk 1]
```diff
@@ -173,6 +174,10 @@ int64_t VCMTiming::RenderTimeMs(uint32_t frame_timestamp,
 
 int64_t VCMTiming::RenderTimeMsInternal(uint32_t frame_timestamp,
                                         int64_t now_ms) const {
+  // For low latency mode, render immediately
+  if (webrtc::field_trial::IsEnabled("OWT-LowLatencyMode")) {
```
Comment (HuaChunbo) [inline modules/video_coding/timing.cc]: constant string should be used here.

File: video/video_receive_stream.cc [hunk 2]
```diff
@@ -82,6 +82,12 @@ VideoCodec CreateDecoderVideoCodec(const VideoReceiveStream::Decoder& decoder) {
 
 namespace internal {
 
+const char kLogLatencyToFileFieldTrial[] = "OWT-Log-Latency-To-File";
```
Comment (HuaChunbo) [inline video/video_receive_stream.cc:85]: Can we share constant string with modules/pacing/paced_sender.cc?

File: modules/pacing/paced_sender.cc [hunk 3]
```diff
@@ -389,12 +401,12 @@ void PacedSender::Process() {
 
   if (!IsLowLatencyMode()) {
     if (packets_->Empty() && !Congested()) {
-      // We can not send padding unless a normal packet has first been sent. If we
-      // do, timestamps get messed up.
+      // We can not send padding unless a normal packet has first been sent. If
```
Comment (HuaChunbo) [inline modules/pacing/paced_sender.cc:404]: We do not

File: modules/video_coding/timing.cc (hunk 1 above)
Comment (taste1981) [reply modules/video_coding/timing.cc]: done.

File: modules/pacing/paced_sender.cc (hunk 3 above)
Comment (taste1981) [reply modules/pacing/paced_sender.cc:404]: done

File: video/video_receive_stream.cc (hunk 2 above)
Comment (taste1981) [reply video/video_receive_stream.cc:85]: done.

**Review by HuaChunbo — APPROVED** [review]

**Pull request merged**